portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,144 @@
 <!DOCTYPE html>
-<html>
+<html "lang"= "en">
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>ElmerAdrianV's Portfolio </title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
+    <script src="script.js"></script>    
+  </head>    
+  
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <section>
+        <h1>A short introduction about me</h1>
+        <img src= "1137.png" alt= "Me in Mexico City" style = "width:240px; height:320px">
+        <p> I'm Elmer Ortega, originally from Matamoros, <abbr title = "Tamaulipas" >Tamps</abbr>, 
+            <abbr title="México"> MX </abbr>. I'm studying a dual bachelor in Applied Mathematics and Computer Engineering 
+            at <a href= "https://itam.mx"> <abbr title = "Instituto 
+                Tecnológico Autónomo de México"> ITAM </abbr> </a>.Currently participate in <a href="https://buildyourfuture.withgoogle.com/programs/softwareproductsprint/">Software Product Sprint by Google</a>.
+        </p>
+        </section>
+
```

Comment:
nit: recommend using the built-in formatter so that indentation, spacing, etc are done for you :) This helps make it more readable

In cloud shell you can right click and select format this document:
![image](https://user-images.githubusercontent.com/34181040/121717581-30f1f300-ca96-11eb-87af-bb2d6445c036.png)